godot-codegen/src/generator/enums.rs

```diff
@@ -251,11 +285,69 @@ fn make_enum_engine_trait_impl(enum_: &Enum) -> TokenStream {
                 fn ord(self) -> i32 {
                     self.ord
                 }
+
+                #str_functions
             }
         }
     }
 }
 
+fn make_enum_str_functions(enum_: &Enum) -> TokenStream {
+    let as_str_enumerators = make_enum_to_str_cases(enum_);
+
+    // Only enumerations with different godot names are specified.
+    // `as_str` is called for the rest of them.
+    let godot_name_enumerators = {
```

Comment:
General question, does this happen a lot? Most Godot enumerators are prefixed with their enum type, while Rust ones aren't :thinking:

Reply:
In terms of differences outside of just the prefix, as far as I know there aren't any. I thought it would be a nice feature to have, but in retrospect maybe it would be better to have a function that just returns the Godot prefix instead? 😅

Reply:
It's not as simple as "Rust ident == Godot prefix + enumerator name", the algorithm to map is quite complex and needs to handle some special cases. As such, a prefix would not be useful.

I checked quickly:
- 4724 enumerators total in the API
- 4565 (96.6%) of which differ between Godot and Rust
- 159 (3.4%) of which remain the same between Godot and Rust

Despite the minority, I think it doesn't hurt much to reuse `as_str()` where possible. If it turns out to be negative in any way, we can always change the implementation.